src/Lean/Exception.lean

```diff
@@ -71,7 +71,7 @@ protected def throwError [Monad m] [MonadError m] (msg : MessageData) : m α :=
 
 /-- Throw an unknown constant error message. -/
 def throwUnknownConstant [Monad m] [MonadError m] (constName : Name) : m α :=
-  Lean.throwError m!"unknown constant '{mkConst constName}'"
+  Lean.throwError m!"unknown constant '{constName}'"
```

Comment:
Should this be using `format`?

```

Reply:
That leads to `unknown constant 'x._@.lean.255._hyg.13'`. Somehow we need to be able to invoke the name sanitizer here...

I think I'm going to put this PR on hold, since `Name` is used for a number of things, and I'm afraid there will be surprises down the line. Maybe better would be to get `{.ofConstName c}` to work and then systematically track down everywhere constants are printed...